src/Client/Pages/Dashboard/DashboardPage.razor.css

```
@@ -0,0 +1,22 @@
+﻿.dashboard-item-box{
+    background-color: var(--prodigo-light-blue);
+    border-radius: 20px;
+    padding: 20px;
+    margin: 10px;
```

nit:
It's more scalable to use `rem` units instead.
```suggestion
    border-radius: 1.25rem;
    padding: 1.25rem;
    margin: 0.625rem;
```

---

🙏